editor/language_server/src/lsp.rs

```diff
@@ -195,6 +201,20 @@ impl LanguageServer for SlideLS {
         self.close(&uri);
     }
 
+    async fn completion(&self, params: CompletionParams) -> Result<Option<CompletionResponse>> {
```

Comment:
result<option> just looks funky, there's got to be something going on thats unneccessary.

Reply:
its required by the trait